### PR TITLE
Add custom title for streaming url

### PR DESCRIPTION
**Pull request by egdbear**
It adds title for currently playing stream or song on lockscreen (iOS) and background process (android).

### CODE REVIEW

**Comment by rafaelteixeira** [comment]
+1

**Comment by justinmc** [comment]
Thanks for this and sorry for the delay!  I didn't know this was possible.  Code looks great at first glance, I'll plan to merge and release a new version tonight when I'm at my computer.

File: src/index.jsx [hunk 1]
```diff
@@ -119,6 +123,7 @@ ReactAudioPlayer.defaultProps = {
   preload: 'metadata',
   src: null,
   style: {},
+  title: ''
```
Comment (justinmc) [inline src/index.jsx]: This should have a trailing comma by the linting rules.  I'm going to go ahead and merge this and fix the commas in a subsequent commit myself.

File: src/index.jsx [hunk 2]
```diff
@@ -141,6 +146,7 @@ ReactAudioPlayer.propTypes = {
   preload: PropTypes.oneOf(['', 'none', 'metadata', 'auto']),
   src: PropTypes.string, // Not required b/c can use <source>
   style: PropTypes.objectOf(PropTypes.string),
+  title: PropTypes.string
```
Comment (justinmc) [inline src/index.jsx]: Another trailing comma needed.

**Comment by justinmc** [comment]
[v0.6.0](https://github.com/justinmc/react-audio-player/releases/tag/v0.6.0) is live on npm with this change.  Thanks again.

**Comment by egdbear** [comment]
@justinmc thanks 👍

**Pull request merged**